src/Native/jitinterface/jithost.cpp

```diff
@@ -48,25 +36,25 @@ class JitHost
     JitHost(JitConfigProvider* pConfigProvider)
         : pConfigProvider(pConfigProvider) { }
 
-    virtual void* allocateMemory(size_t size, bool usePageAllocator = false)
+    virtual void* DEFAULT_CALL_CONV allocateMemory(size_t size, bool usePageAllocator = false)
```

Comment:
I think this should stay as is, without explicit callconv marker.

It does not have any call conv in CoreCLR, so it will use the real default. The real default on Windows x86 for instance call like this is `thiscall` that is different from `stdcall`.